tracer/src/Datadog.InstrumentedAssemblyGenerator/Datadog.InstrumentedAssemblyGenerator.csproj

```
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <Company>Datadog Ltd.</Company>
+    <Copyright>Datadog Ltd.</Copyright>
```

nit - To be coherent with other occurences in the repo
```suggestion
    <Company>Datadog</Company>
    <Copyright>Copyright 2022 Datadog, Inc.</Copyright>
```

---

Fixed in 4c7c44dcf9bc4f5affbde6c20b8fd17d1735d225